### PR TITLE
feature searchbar works

### PR DESCRIPTION
**Pull request by axeljdev**
feature searchbar works, css work in progress

### CODE REVIEW

File: client/src/components/SearchBar.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  }
+
+  const handleChange = (e) => {
+    const searchValue = e.target.value;
+      getMovie(`https://api.themoviedb.org/3/search/multi?language=fr-FR&query=${searchValue}`)
+  }
+
+  return (
+    <>
+      <input type="search" placeholder="Rechercher" onChange={handleChange} />
```
Comment (Yoann-Fortin) [inline client/src/components/SearchBar.jsx:30]: label and link is missing

File: client/src/components/SearchBar.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <section>
+      {movies.map(movie => (
+        <article key={movie.id}>
+          <figure>
+            <img src={`https://image.tmdb.org/t/p/original/${movie.poster_path}`} alt={movie.title}/>
+          </figure>
+        </article>
```
Comment (Yoann-Fortin) [inline client/src/components/SearchBar.jsx:37]: title in figcaption will be better

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Review by Sylvainlfe — APPROVED** [review]

**Review by Alice-Berthelot — APPROVED** [review]

File: client/src/components/SearchBar.jsx (hunk 2 above)
Comment (axeljdev) [reply client/src/components/SearchBar.jsx]: fixed

File: client/src/components/SearchBar.jsx (hunk 1 above)
Comment (axeljdev) [reply client/src/components/SearchBar.jsx]: fixed

**Pull request merged**